packages/calypso-e2e/src/lib/pages/site-editor-page.ts

```diff
@@ -0,0 +1,170 @@
+import { Locator, Page } from 'playwright';
+import {
+	BlockInserter,
+	EditorGutenbergComponent,
+	EditorSidebarBlockInserterComponent,
+	EditorToolbarComponent,
+	EditorWelcomeTourComponent,
+} from '..';
+import { getCalypsoURL } from '../../data-helper';
+import envVariables from '../../env-variables';
+
+const wpAdminPath = 'wp-admin/themes.php';
+
+const selectors = {
+	editorIframe: `iframe.is-loaded[src*="${ wpAdminPath }"]`,
```

Comment:
I'm a bit astounded that the `src` is needed to select the editor iframe for FSE...

Reply:
It isn't _completely_ necessary, but a partial match seemed like the most stable, unique distinction of the iframes. The core path is unique and won't change over time!

I can update it too something else though if you prefer! 👍

Reply:
I don't mind it per se, though it does feel like it adds an odd dependency on some other thing.

Reply:
I went and double-checked, and unfortunately, there's nothing else on the iframe that we can really use other than `.is-loaded`, which feels risky to use exclusively because there are other iframes we have to worry about. 😞 

A `name` attribute would have been so nice!!